.github/workflows/tests.yaml

```
@@ -10,22 +10,26 @@ jobs:
       - uses: actions/checkout@v3
       - name: Trunk Check
         uses: trunk-io/trunk-action@v1
-  tests:
+  test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        job-type: [unit, api]
```

Not a job type but a test type I guess
```suggestion
        test: [unit, api]
```